web/core/store/issue/issue-details/issue.store.ts

```diff
@@ -318,34 +321,34 @@ export class IssueStore implements IIssueStore {
     this.rootIssueDetailStore.rootIssueStore.issues.addIssueIdentifier(issueIdentifier, issueId);
```

Comment:
Let's use `rootWorkItemDetailStore` here as well to access `rootIssueStore` for consistency.